docs/source/guides/dfs_with_dask_entitysets.rst

```diff
@@ -0,0 +1,102 @@
+
+.. ipython:: python
+
+    import featuretools as ft
+    import pandas as pd
+    import dask.dataframe as dd
+    id = [0, 1, 2, 3, 4]
+    values = [12, -35, 14, 103, -51]
+    df = pd.DataFrame({"id": id, "values": values})
+    dask_df = dd.from_pandas(df, npartitions=2)
+    dask_df
+
+
+Now that we have our Dask dataframe, we can start to create our entityset. The current implementation does not support variable type inference for Dask entities, so we must pass a dictionary of variable types using the ``variable_types`` parameter when calling ``es.entity_from_dataframe()``. Aside from needing to supply the variable types, the rest of the process of creating an entityset is the same as if we were using pandas dataframes.
+
+.. ipython:: python
+
+    es = ft.EntitySet(id="dask_es")
+    es = es.entity_from_dataframe(entity_id="dask_entity",
+                                  dataframe=dask_df,
+                                  index="id",
+                                  variable_types={"id": ft.variable_types.Id,
+                                                  "values": ft.variable_types.Numeric})
+    es
+
+
+Notice that when we print our entityset, the number of rows for the ``dask_entity`` entity is returned as a Dask ``Delayed`` object. This is because obtaining the length of a Dask dataframe requires an expensive compute operation to sum up the lengths of all the individual partitions that make up the dataframe and that operation is not performed by default.
+
+
+Running DFS
+-----------
+We can pass the entityset we created above to ``featuretools.dfs`` in order to create a feature matrix. If the entityset we pass to ``dfs`` is made of Dask entities, the feature matrix we get back will be a Dask dataframe.
+
+.. ipython:: python
+
+    feature_matrix, features = ft.dfs(entityset=es,
+                                      target_entity="dask_entity",
+                                      trans_primitives=["negate"])
+    feature_matrix
+
+
+This feature matrix can be saved to disk or computed and brought into memory, using the appropriate Dask dataframe methods.
+
+.. ipython:: python
+
+    fm_computed = feature_matrix.compute()
+    fm_computed
+
+
+While this is a simple example to illustrate the process of using Dask dataframes with Featuretools, this process will also work with entitysets containing multiple entities, as well as with aggregation primitives.
+
+Limitations
+-----------
+There are many parts of Featuretools that are difficult to implement in a distributed environment and several primitives that are not well suited to operate on distributed dataframes. As a result, there are some limitations when creating Dask entitysets and then using these entitysets to generate a feature matrix. The most significant limitations are reviewed in more detail in this section.
+
+Supported Primitives
```

Comment:
Should we explicitly state the types of primitive computations that are unsupported? Something like:
Primitives that rely on the order of the entire dataframe or require an entire column to be computed are inefficient and difficult to support in a distributed environment.